src/components/sections/EventContactOrganizer.astro

```diff
@@ -0,0 +1,37 @@
+---
+import phone from "../../../public/phone.svg?raw";
+import mail from "../../../public/fi_mail.svg?raw";
+---
+
+<section class="flex justify-between items-center sm:w-[1142px] w-[375px] mx-4 my-16">
+    <div class="w-[704px] sm:ml-[149px]">
```

Comment:
I see so many custom pixel values here, we should considering using percentage instead for more responsiveness